.github/scripts/sink_cache.py

```diff
@@ -0,0 +1,235 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+# CLI to Delete Github Action Cache.
+#
+# This script remove Cache entries a specified Github repository.
+# This allow future workflow to run on clear cache data (since they'll have to recreate them)
+import argparse
+from datetime import datetime
+from functools import reduce
+import json
+import logging
+import subprocess
+from typing import Any, Dict, Optional
+from logging import basicConfig, getLogger
+
+import sys
+from os.path import dirname, abspath
+
+current_dir = dirname(abspath(__file__))
```

Comment:
pathlib is a much better module when dealing with path:
```python
from pathlib import Path
current_dir = Path(__file__).resolve().parent
```

Reply:
I've remove the dependency, so I also removed this code block